pkg/bundle/bundle_test.go

```diff
@@ -141,12 +141,21 @@ func Test_Reconcile(t *testing.T) {
 			Bundle:  dummy.TestCertificate5,
 		}
 
-		configMapPatch = func(name, namespace string, data map[string]string, binData map[string][]byte, key *string) *coreapplyconfig.ConfigMapApplyConfiguration {
+		configMapPatch = func(name, namespace string, data map[string]string, binData map[string][]byte, key *string, additionaFormats *trustapi.AdditionalFormats) *coreapplyconfig.ConfigMapApplyConfiguration {
 			annotations := map[string]string{}
 			if key != nil {
 				annotations[trustapi.BundleHashAnnotationKey] = fmt.Sprintf("%x", sha256.Sum256([]byte(data[*key])))
 			}
 
+			if additionaFormats != nil {
```

Comment:
Can we use "truststorePasswordAnnotations" here?

Reply:
As this particular variable of type `trustapi.AdditionalFormats` I guess it is good to keep a name which reflects its designation.

```
configMapPatch = func(name, namespace string, data map[string]string, binData map[string][]byte, key *string, additionaFormats *trustapi.AdditionalFormats) *coreapplyconfig.ConfigMapApplyConfiguration {
			annotations := map[string]string{}
			if key != nil {
				annotations[trustapi.BundleHashAnnotationKey] = fmt.Sprintf("%x", sha256.Sum256([]byte(data[*key])))
			}

			if additionaFormats != nil {
				if additionaFormats.JKS != nil {
...
```

Reply:
I guess it leads to a confusion, as `annotation` variable consist hashes not for passwords only, there is annotation for `trustapi.BundleHashAnnotationKey` as well. 
So, I would keep the name as it is.

Reply:
Sorry, I was unclear. What I meant to ask is if we can use the existing function "truststorePasswordAnnotations" here instead of the logic that follows.

Reply:
@erikgb asked me to do use logic statements instead of function

Reply:
@arsenalzp, my comment was addressing the `needsUpdate` logic, and not the patch assert here in the test..

Reply:
Hello @erikgb @inteon,
Any objections regarding my statement in previous message?

Reply:
I am planning to review this PR again, but my week has been busy so far.

Reply:
I must say I agree with @arsenalzp on this. Exporting the internal function seems wrong to me too. It might be my previous refactoring that complicates this a bit, and I can try to improve this in a follow-up PR.